app/interactors/donation/create.rb

```diff
@@ -0,0 +1,17 @@
+class Donation
+  include Charge
+  include Customer
+  include Token
+
+  class Create
+    include Interactor::Organizer
+
+    organize Token::Generate, Customer::Create, Charge::Create
```

Comment:
In such case calling `fail!` method prior to `raise ActiveRecord::Rollback` is sufficient enough